examples/api/lib/cupertino/modal_bottom_sheet/modal_bottom_sheet.1.dart

```diff
@@ -0,0 +1,478 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
```

Comment:
This one also should have a description comment here I think.

Reply:
I'm not sure what the difference is between this example and the first one, so this is probably a good place to explain that.